### PR TITLE
Adds new integration [dcmeglio/homeassistant-waste_management]

### PR DESCRIPTION
**Pull request by dcmeglio**
**Link to successful HACS action:**   https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3809979760
**Link to successful hassfest action (if integration):**  https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3809979759

### CODE REVIEW

File: integration [hunk 1]
```diff
@@ -212,6 +212,7 @@
   "davesmeghead/visonic",
   "dckiller51/bodymiscale",
   "dcmeglio/homeassistant-petsafe",
+  "dcmeglio/homeassistant-waste_management",
```
Comment (ludeeus) [inline integration:215]: This is impossible to use.
There are nothing that show which service/which country
It requires user/pass, but without knowing to what it just cant be used by anyone (other than the author)

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: integration (hunk 1 above)
Comment (dcmeglio) [reply integration:215]: I believe you're thinking this is some generic name, but Waste Management is the name of the company. It's the largest garbage removal company in the world serving most of North America. The integration includes their logo too.  They have 30 million customers so I think it's useful to others aside from me, https://www.wm.com/

Comment (ludeeus) [reply integration:215]: A link should be added as a minimum.
I did a google search for “waste management” and got multiple hits for every country in the world

Comment (ludeeus) [reply integration:215]: And it needs a list of countries in hacs.json

Comment (dcmeglio) [reply integration:215]: I updated the repo description to "Determines your next Waste Management (https://www.wm.com/) pickup times for use in Home Assistant." so hopefully that avoids confusion. Added the countries listed on their website to the hacs.json, https://github.com/dcmeglio/homeassistant-waste_management/blob/master/hacs.json

Latest HACS run, https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3920462676
Latest hassfest run, https://github.com/dcmeglio/homeassistant-waste_management/actions/runs/3920462674

Let me know if anything else is needed

**Pull request merged**